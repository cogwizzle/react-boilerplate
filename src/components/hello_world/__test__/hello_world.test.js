import * as React from 'react';
import HelloWorld from '../hello_world';
import * as renderer from 'react-test-renderer';

it('Render', () => {
  const component = renderer.create(
    <HelloWorld />
  );

  const results = component.toJSON();
  expect(results).toMatchSnapshot();
});

