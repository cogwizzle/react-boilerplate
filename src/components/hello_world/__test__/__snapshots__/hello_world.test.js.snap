// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render 1`] = `
<div
  className="hello_world"
>
  Hello World
  <span
    className="hello_world__sub"
  >
    !
  </span>
</div>
`;
